luarules/gadgets/unit_builder_priority.lua

```diff
@@ -311,17 +464,25 @@ function gadget:TeamChanged(teamID)
 end
 
 function gadget:GameFrame(n)
-    for builderID, builtUnit in pairs(buildTargetOwners) do
-        if spValidUnitID(builderID) and spGetUnitIsBuilding(builderID) == builtUnit then
-            spSetUnitBuildSpeed(builderID, currentBuildSpeed[builderID])
-        end
-    end
-	buildTargetOwners = {}
-    buildTargets = {}
-
+	
+	-- Thus on the next frame, we can loop through build target owners and
```

Comment:
"Thus" feels like there's more to this explanation that's not present

Reply:
Leftover comment :p